local-notifications/android/src/main/java/com/capacitorjs/plugins/localnotifications/LocalNotificationsPlugin.java

```diff
@@ -44,12 +44,6 @@ protected void handleOnNewIntent(Intent data) {
         }
     }
 
-    @Override
-    protected void handleOnActivityResult(PluginCall savedCall, int requestCode, int resultCode, Intent data) {
```

Comment:
@dwieeb when I was updating this plugin it looks like this result callback wasn't used. I looked into it but couldn't see where this might be reached. Since you looked over this code recently, do you know if that's true? I think this is safe to delete.

Reply:
Yeah, it should be.